test/libsolidity/GasCosts.cpp

```diff
@@ -104,9 +104,19 @@ BOOST_AUTO_TEST_CASE(string_storage)
 	else if (Options::get().useABIEncoderV2)
 	{
 		if (Options::get().optimizeYul)
-			CHECK_DEPLOY_GAS(151455, 127653, evmVersion);
+		{
+			if (evmVersion < EVMVersion::istanbul())
```

Comment:
It might have been better to directly take the data gas cost change into account and keep the constants.